DataminrPulse/resources/logs/DataminrPulse.txt

```diff
@@ -0,0 +1,4 @@
inks":[]},{"id":" \"CVE-2022-29404\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2022-26377\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2022-28330\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2022-37436\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2006-20001\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2022-36760\"","cvss":0.0,"products":[],"exploitPocLinks":[]},{"id":" \"CVE-2023-25690\"]","cvss":0.0,"products":[],"exploitPocLinks":[]}],"URLs":["konzern.porr.at"],"addresses":[{"ip":"195.216.229.6","port":0,"version":""}],"asns":["AS47186"],"orgs":["PORR AG"],"hashes":[],"products":["Apache httpd"],"malwares":[],"threats":[],"asOrgs":[{"asn":"AS47186","asOrg":"PORR AG"}],"hashValues":[]}}],"targets":[]},"location":{"coordinates":[48.1521,16.3878],"latitude":48.1521,"longitude":16.3878,"name":"Vienna, Austria","places":["db828c7ed4154b5daf9db0998abba26a","57cd565071524e68abaa18657677721b","4687a3f12b814d5c5b87e5268f2195e3","4172b782a3184f63a847f065c53768f9","d46e190f5e05c5ebe64c9630e8ea985a","65b9491275ec4fed596b6dcc831fb381"],"probability":0.0,"radius":0.1},"metadata":{},"watchlistsMatchedByType":[{"id":"3999781","type":"topics","name":"Cyber Alerts","externalTopicIds":["961954"],"userProperties":{"uiListType":"CYBER","omnilist":"true"}}],"topics":[{"name":"PORR AG","topicType":"company","id":"e323fc065bb23ab8fde43d2504524cdd","idStr":"e323fc065bb23ab8fde43d2504524cdd","retired":false}],"categories":[{"name":"Email and Web Servers","topicType":"category","id":"961954","idStr":"961954","path":"/TOPIC/EXT/CS/961954","retired":false,"requested":"true"},{"name":"Cybersecurity - Threats & Vulnerabilities","topicType":"category","id":"853086","idStr":"853086","path":"/TOPIC/EXT/CS/853086","retired":false,"requested":"false"},{"name":"Cybersecurity","topicType":"category","id":"124022","idStr":"124022","path":"/TOPIC/EXT/CS/124022","retired":false,"requested":"false"}],"sectors":[],"publisherCategory":{"id":"chatter","name":"Chatter"},"expandAlertUrl":"https://app.dataminr.com/#alertDetail/5/5097682483652307508-1697109988162-1","headline":"PORR AG IP 195.216.229.6 hosts server configuration with CVE(s) including Apache HTTP Server-Side Request Forgery (SSRF) CVE-2021-40438: Sensor via Shodan.","availableRelatedAlerts":0,"odsStatus":{"type":"Shodan","timestamp":1697109988162,"languages":[],"source":{"id":"","channels":["sensor"],"id_str":""},"link":"https://www.shodan.io/host/3285771526","media":[],"extraData":{"displaySourceName":"Shodan"}},"referenceTerms":[{"term":"server vulnerabilities"},{"term":"ssrf"},{"term":"forgery"},{"term":"cve"}],"watchlistsMatched":["3999781"],"alertType":{"id":"alert","name":"Alert"},"companies":[{"name":"PORR AG","topicType":"company","id":"e323fc065bb23ab8fde43d2504524cdd","idStr":"e323fc065bb23ab8fde43d2504524cdd","retired":false}],"image_url":null,"source_type":["sensor"],"source_link":"https://www.shodan.io/host/3285771526"}
```

Comment:
I was able to test the dashboards using the events in this txt file but found that most of the panels in Dataminr - Cyber Threat Landscape dashboard are not populating could you please provide sample events for this dashboard.
<img width="1322" alt="image" src="https://github.com/SumoLogic/sumologic-public-partner-apps/assets/3620468/91f1f8dc-6b90-46e9-bb46-bf3b5d75ded0">

Reply:
Added respective logs to the txt file.